helm/values.yaml

```diff
@@ -136,5 +136,5 @@ env:
   REDIS_HOST: "localhost"
   REDIS_PORT: "6379"
   AUTO_REINDEX_INTERVAL: "5400"
-  SNAPSHOT: "{{ .Chart.AppVersion }}"
```

Comment:
BibXML uses Snapshot to show app version.
So this should get the `{{ .Chart.AppVersion }}` because that gets dynamically updated when GHA publish the chart.